packages/transaction-controller/src/TransactionController.ts

```diff
@@ -587,23 +590,22 @@ export class TransactionController extends BaseController<
    * specified in txParams, @ethereumjs/tx is able to determine which EIP-2718
    * transaction type to use.
    *
-   * @returns {Common} common configuration object
+   * @returns Common configuration object.
    */
-
   getCommonConfiguration(): Common {
     const {
       network: networkId,
       providerConfig: { type: chain, chainId, nickname: name },
-    } = this.getNetworkState();
+    } = this.#getNetworkState();
 
     if (chain !== RPC) {
       return new Common({ chain, hardfork: HARDFORK });
     }
 
     const customChainParams = {
       name,
-      chainId: parseInt(chainId, undefined),
-      networkId: parseInt(networkId, undefined),
+      chainId: parseInt(chainId, NaN),
+      networkId: parseInt(networkId, NaN),
```

Comment:
Is `NaN` appropriate here? Should this be 16 or 10?

Reply:
I'm not sure. `NaN` is the equivalent of `undefined`, so that's why I used that.

Reply:
Interesting... I did not know that.